manifests/s/SFTPGo/SFTPGoEnterprise/2.7.2025.1107/SFTPGo.SFTPGoEnterprise.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.10.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: SFTPGo.SFTPGoEnterprise
```

Comment:
Is there a reason this isn't under `drakkan.SFTPGo.Enterprise` so that it's near the original `drakkan.SFTPGo` ?